### DISCUSSION TITLE
Announcing SSKR now available on all Ledger Nano devices

### DISCUSSION THREAD

**Discussion by aido** [BlockchainCommons/Gordian-Developer-Community#131] [General & Announcements]
I am happy to announce that there is now a tool available to over **7 million** hardware wallets that enables the protection of assets with SSKR.

The Ledger Seed Tool app (app-seed-tool) has been released and [v1.7.4](https://github.com/aido/app-seed-tool/releases/tag/v1.7.4) can now be installed from Ledger Live.

The app is available for all Ledger Nano devices; Nano S, Nano S Plus and Nano X.
The Ledger Stax version is still a work in progress and the next version of the app will include all the screens needed to generate SSKR on Stax too.
The bigger e-ink screen on the Stax devices also gives the potential to implement some other interoperable specifications like animated QR, Envelope and UR.

If you would like to see some screenshots or a short animation of the various menus then they are available here:
[Screenshots](https://github.com/aido/app-seed-tool/tree/screenshots/tests/functional/screenshots)

A blogpost will soon be released on the Ledger website alongside a Ledger Help Centre article redirecting to it and mentioning that Stax support is still a work in progress.

**Comment by ChristopherA** [comment]
Great to hear!

Would you be up to a 15m presentation (and maybe demo if feasible) at our next Gordian Meeting on August 7th at 10am PDT? Send me mail at ChristopherA@LifeWithAlacrity.com if interested.

**Reply by aido** [reply]
Hi @ChristopherA ,

Sure.
The date suits and the time also suits the time zone I'm in.

I don't have a slide deck prepared so not sure how 'professional' a presentation will be. But I do have plenty of screen shots of the app on Nano S and Stax devices stored on GitHub and I can talk through them as part of a demo, The README on GitHub also has a flowchart and descriptions I can use as an part of an explainer too.

As mentioned, Ledger should have their blog post and :Ledger Help Centre article published by then. I don't yet know what that is going to look like but it may be helpful for a presentation also.

Would that suffice?

**Reply by ChristopherA** [reply]
Perfect! 👍

**Reply by aido** [reply]
Great.

I'll firm up on my attendance closer to the date,

**Comment by aido** [comment]
The Ledger blog post on app-seed-tool is now available at https://www.ledger.com/blog/seed-tool-app
